app/controllers/static_pages_controller.rb

```diff
@@ -1,5 +1,10 @@
 class StaticPagesController < ApplicationController
-  def home; end
+  def home
+    return unless logged_in?
+    @micropost  = current_user.microposts.build
+    @feed_items = current_user.microposts.page(params[:page]).
+      per(Settings.per_page_user)
```

Comment:
em ngắt dòng ở trước dấu chấm nhé, chứ ko phải phía sau nó như thế này:
```suggestion
      .per(Settings.per_page_user)
```